src/selectedtests/project_config.py

```diff
@@ -0,0 +1,195 @@
+"""Domain object representing project config."""
+from pymongo.collection import Collection
+
+
+class TaskConfig:
+    """Represents the task mappings config for a project config."""
+
+    def __init__(
+        self,
+        most_recent_version_analyzed=None,
+        source_file_regex=None,
+        build_variant_regex=None,
+        module=None,
+        module_source_file_regex=None,
+    ):
+        """Init a TaskConfig instance. Use ProjectConfig.get rather than this directly."""
+        self.most_recent_version_analyzed = most_recent_version_analyzed
+        self.source_file_regex = source_file_regex
+        self.build_variant_regex = build_variant_regex
+        self.module = module
+        self.module_source_file_regex = module_source_file_regex
+
+    @classmethod
+    def from_json(cls, json):
+        """
+        Instantiate an instance of TaskConfig from json. Use ProjectConfig.get instead.
+
+        :param json: Json representing project task config in database.
+        :return: An instance of TaskConfig.
+        """
+        return cls(
+            json.get("most_recent_version_analyzed", None),
```

Comment:
[nit] I believe if the given key is not found in the dictionary, `.get` will return `None` if no default is specified. So I don't think you need the 2nd parameter if it is `None`.